src/utils/orion_utils.cairo

```diff
@@ -28,6 +28,22 @@ mod OrionUtils {
         }
     }
 
+    impl i32TryIntou32 of TryInto<i32, u32> {
+        fn try_into(self: i32) -> Option<u32> {
+            assert(self.sign == false, 'The sign must be positive');
+            let max: u32 = BoundedInt::max();
+            assert(self.mag <= max.into(), 'Overflow of magnitude');
```

Comment:
could you add a test case for this assert?

Reply:
Now that I see it, this assert should not exist because it will never happen. With i32, the maximum positive value we can represent is 2^32 / 2 - 1 (using the new method), and the maximum for u32 is 2^32. Therefore, that assert doesn't make sense.